inference/worker/interface.py

```diff
@@ -33,20 +45,43 @@ def from_work_parameters(params: inference.WorkParameters) -> "GenerateStreamPar
 
 
 class GenerateStreamRequest(pydantic.BaseModel):
+    """
+    Request to generate a stream.
```

Comment:
I think if we are going to add docstrings to these we should try to make them a bit more informative than just restating what can already be inferred from the name, to be honest